src/LibraryManager/Manifest.cs

```diff
@@ -111,6 +122,11 @@ public static Manifest FromJson(string json, IDependencies dependencies)
             }
         }
 
+        internal void AddLibraryValidator(LibrariesValidator librariesValidator)
```

Comment:
>AddLibraryValidator [](start = 22, length = 19)

This is a test hack.  Can we make it naturally work, like with a `LibraryValidatorFactory` that we can pass to `FromJson`?